Project.toml

```
@@ -16,7 +16,7 @@ Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
 
 [compat]
 Compat = "2, 3"
-DSP = "0.6.1, 0.7"
+DSP = "0.6.1, 0.7.1"
```

Does this imply that `0.7.0, 0.7.2, 0.7.3 etc` dont work?

---

No, it implies that all versions [0.7.1, 0.8.0) are compatible. See here for more info: [Behavior of versions with leading zeros](https://pkgdocs.julialang.org/v1/compatibility/#compat-pre-1.0)

---

Would that then drop support for `0.7.0`?
Would this be better?
```suggestion
DSP = "0.6.1 - 0.7"
```
Which is equivelent to [0.6.1, 0.8.0)

---

That's a good suggestion @rob-luke. I'll update it.